content/docs/start/experiments.md

```diff
@@ -18,17 +18,90 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section
+we'll use a
+[Get Started with Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0.
+
+<details>
+
+### ⚙️ Click here for the installation instructions
+
+These commands are run in the
+[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
+project. You can run these commands after cloning the repository and installing
+the requirements.
+
+Please clone and create a virtual environment:
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments
+$ cd get-started-experiments
+$ python -m venv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
```

Comment:
Let's just mention/link this as a note after the code block, similar to the **Expand to download example code** block in https://dvc.org/doc/start/data-pipelines#pipeline-stages:

![image](https://user-images.githubusercontent.com/1477535/120762506-f0193d80-c4db-11eb-914f-12728e4b21a5.png)

Reply:
BTW https://github.com/iterative/get-started-experiments/issues/2

Reply:
I replaced `python -m venv` with `virtualenv`